npm/pkg/dataplane/ipsets/ipsetmanager_windows.go

```diff
@@ -327,6 +329,11 @@ func getPolicyNetworkRequestMarshal(setPolicySettings map[string]*hcn.SetPolicyS
 		)
 	}
 
+	if len(policyNetworkRequest.Policies) == 0 {
```

Comment:
why move this check down ?

Reply:
Added the other check back, but this is a separate check. We can have 1+ setpolicysettings but have 0 policies if there are no setpolicies with the given policyType